src/components/VideoPlayer.vue

```diff
@@ -2,11 +2,18 @@
     <div
         ref="container"
         data-shaka-player-container
-        class="w-full max-h-screen flex justify-center"
+        class="w-full max-h-screen flex justify-center relative"
         :class="{ 'player-container': !isEmbed }"
     >
         <video ref="videoEl" class="w-full" data-shaka-player :autoplay="shouldAutoPlay" :loop="selectedAutoLoop" />
-        <canvas id="preview" />
+        <span
+            ref="previewContainer"
+            id="preview-container"
+            class="hidden flex-col absolute bottom-0 z-[2000] mb-[4.5%] items-center"
```

Comment:
I find that `mb-[3.5%]` looks better on my screen. I guess we need to test this on different view ports :thinking: